scripts/v.dissolve/v.dissolve.html

```diff
@@ -75,8 +75,3 @@ <h2>AUTHORS</h2>
 module: M. Hamish Bowman, Dept. Marine Science, Otago University, New Zealand<br>
 Markus Neteler for column support<br>
 help page: Trevor Wiens
-
-<!--
-<p>
-<i>Last changed: $Date$</i>
--->
```

Comment:
Line 10, 28, 62, 72 unnecessary one empty line.

Reply:
This is way beyond the original idea of this PR... I tend to leave it as is, this PR is already too large for me.

Perhaps someone is willing to write some `sed` (or whatever) magic to streamline it?
Or someone else may update this PR accordingly.

So far we mostly have two empty lines before `<h2>` for readability purposes. I am fine to see that dropped but the original intention of this PR was to get rid of the `SVN Date` comments.

Reply:
Ok, I undestand. I saw that other issues are solved in previous reviews, I did not want to overwhelm you, you can ignore it.

Reply:
> So far we mostly have two empty lines before `<h2>` for readability purposes. I am fine to see that dropped but the original intention of this PR was to get rid of the `SVN Date` comments.

Yes, I thought so, but in most cases there is only one blank line.

Reply:
> This is way beyond the original idea of this PR...

I agree the PR is too large already. These large PRs need to be "monothematic".

> ...some...magic to streamline it?

A Bash script with _sed_ or a Python script with a series of regexps (for anything more complicated) could run in the CI to check for these things. (This would follow the goal of having only rules we can automatically enforce.)

Reply:
@tmszi so, in the end I have done it, with `geany` :-D - see 5c09ae064f115c28c72efa6e087e0eba7a8a124f
It wasn't too hard in the end doing bulk modifications.